haproxy/Dockerfile

```diff
@@ -5,19 +5,21 @@ RUN apt-get update \
  && apt-get install -y --no-install-recommends \
     ca-certificates \
     python3-pip \
+    cron \
+    rsyslog \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* \
- && pip3 install chaperone \
  && groupadd -r haproxy \
  && useradd -r -g haproxy haproxy \
  && mkdir -p /var/log/haproxy \
  && ln -s /usr/local/etc/haproxy /etc/haproxy \
- && chown -R haproxy:haproxy /usr/local/etc/haproxy /var/log/haproxy
+ && chown -R haproxy:haproxy /usr/local/etc/haproxy /var/log/haproxy \
+ && sed -i '/#cron./c\cron.*                          \/proc\/1\/fd\/1'  /etc/rsyslog.conf
+
 
-COPY src/chaperone.conf         /etc/chaperone.d/chaperone.conf
 COPY src/reload                 /usr/bin/
 COPY src/haproxy.cfg            /tmp/
 COPY src/docker-setup.sh src/configure.py src/track_hosts src/track_dns /
 
-ENTRYPOINT ["/usr/local/bin/chaperone"]
-CMD ["--user", "haproxy"]
+ENTRYPOINT ["/docker-setup.sh"]
```

Comment:
Please rename to `/docker-entrypoint.sh`. 

You'll also need to move original `/docker-entrypoint.sh` to `haproxy-entrypoint.sh` and then you can remove explicit `ENTRYPOINT` from Dockerfile. Then you'll have to call `haproxy-entrypoint.sh` from your `docker-entrypoint.sh`

Reply:
We don't use the original /docker-entrypoint.sh, I have incorporated the code in docker-setup.sh